readthedocs/projects/models.py

```diff
@@ -144,10 +144,11 @@ class AddonsConfig(TimeStampedModel):
     Everything is enabled by default.
     """
 
-    DOC_DIFF_DEFAULT_ROOT_SELECTOR = "[role=main]"
-    LINKPREVIEWS_DEFAULT_ROOT_SELECTOR = "[role=main] a.internal"
-    LINKPREVIEWS_DOCTOOL_NAME_CHOICES = (
+    DEFAULT_ROOT_SELECTOR = ":is([role=main],main,div.body,div.document)"
+    DOCTOOL_NAME_CHOICES = (
```

Comment:
Seems like we're going to want more of these over time, and some way to have users input one, but we can start with this for now I guess.